lstchain/mc/sensitivity.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+import pandas as pd
+import astropy.units as u
+
+from eventio.simtel.simtelfile import SimTelFile
+
+from .plot_utils import sens_plot, sens_minimization_plot
+from .mc import rate, weight
+from lstchain.spectra.crab import crab_hegra
+from lstchain.spectra.proton import proton_bess
+
+
+def read_sim_par(source):
+    """
+    Read MC simulated parameters
+
+    Parameters
+    ---------
+    source: simtelarray file
+
+    Returns
+    ---------
+    par: `dict` with simulated parameters
+
+    """
+    emin, emax = source.mc_run_headers[0]['E_range'] * u.TeV
+    sp_idx = source.mc_run_headers[0]['spectral_index']
+    n_showers = source.mc_run_headers[0]['num_showers']
+    n_use = source.mc_run_headers[0]['num_use']
+    sim_ev= n_showers * n_use
+    max_impact = source.mc_run_headers[0]['core_range'][1] * u.m 
+    area_sim = np.pi * np.power(max_impact,2)
+    cone = source.mc_run_headers[0]['viewcone'][1] * u.deg
+
+    par_var = [emin, emax, sp_idx, sim_ev, area_sim, cone]
+    par_dic = ['emin', 'emax', 'sp_idx', 'sim_ev', 'area_sim', 'cone']
+    par = dict(zip(par_dic, par_var))
+
+    return par
+
+def process_mc(simtelfile, dl2_file):
+    """
+    Process the MC simulated and triggered to extract the relevant
```

Comment:
I thought I commented this already but can't seem to find it...?
DL2 files include **reconstructed** events.

Reply:
But we need the original information, that is why we are also reading the `simtelfile`, is this what you mean?

Reply:
Or maybe I misunderstand the docstring.
When I read this, I think "MC simulated" applies to the `simtelfile` and "triggered" applies to the `dl2_file`

Reply:
yes, that is right, it was probably a mistake from my side to call it like this... I changed triggered by reconstructed in all the namings